tests/unit/obuf-limits.tcl

```diff
@@ -145,6 +146,9 @@ start_server {tags {"obuf-limits"}} {
         r set k3 v3
         r del k1
         catch {[r exec]} e
+        assert_match "*I/O error*" $e
+        set clients [r client list]
+        assert_no_match "*name=transactionclient*" $clients
         reconnect
```

Comment:
```suggestion
        assert_match "*I/O error*" $e
        reconnect
        set clients [r client list]
        assert_no_match "*name=transactionclient*" $clients
```
i rather not rely on the implicit reconnect, and issue all the rest of the commands after an explicit one.